test/JDBC/input/case_insensitive_collation-vu-verify.sql

```diff
@@ -0,0 +1,40 @@
+select name from sys.columns where name = 'Test';
+go
+
+select name from sys.columns where name = 'TesT';
+go
+
+select COUNT(*) FROM sys.tables WHERE name = 'cameltable';
+go
+
+select COUNT(*) FROM sys.tables WHERE name = 'CAMELTABLE';
+go
+
+select COUNT(*) FROM sys.tables WHERE name = 'CaMeLtAbLe';
+go
+
+SELECT COUNT(*) FROM sys.columns WHERE name = 'cAmElCoLuMn';
+go
+
+select COUNT(*) FROM sys.views WHERE name = 'CAMELview';
+go
+
+select name from sys.databases where name = 'test';
+go
+
+select name from sys.databases where name = 'TEST';
+go
+
+select * from vB;
+go
+
+SELECT t.TABLE_NAME
+FROM INFORMATION_SCHEMA.TABLES AS t
+JOIN sys.objects o ON t.TABLE_NAME = o.name where o.name = 'mytable' AND o.object_id != 0;
```

Comment:
if entry is present in INFORMATION_SCHEMA.TABLES then object_id would alway be non zero. So we do not need to have `AND o.object_id != 0`

Reply:
removed